### DISCUSSION TITLE
Kingdom Rush - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#18] [General]
# Kingdom Rush - Alan Pope's blog

Last week I posted about my guilty collection of unplayed games in Digital Hoarding: Gaming Edition. In short, I have a ton of games I’ve bought over the years and never played, even once. I set myself an internal goal to play more of the games I already have, and reign in my game purchasing.
I used the dynamic collection feature of Steam to show me the unplayed games, sorted by their steam review score.

https://popey.com/blog/2021/01/kingdom-rush/

**Comment by JupiterSky11** [comment]
I played through this game on mobile a long time ago and can attest to its quality.  A very fun game to play, would probably be good to put it on my steam deck.
